front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3310,7 +3311,98 @@ export type TrainSchedule = {
   start_time: string;
   train_name: string;
 };
+export type CategoryChangeGroup = {
+  value: string;
+};
+export type ConstraintDistributionChangeGroup = {
+  value: Distribution;
+};
+export type InitialSpeedChangeGroup = {
+  value: number;
+};
+export type LabelsChangeGroup = {
+  value: string[];
+};
+export type TrainScheduleOptions = {
+  use_electrical_profiles?: boolean;
+  use_speed_limits_for_simulation?: boolean;
+};
+export type OptionsChangeGroup = {
+  value: TrainScheduleOptions;
+};
+export type Margins = {
+  boundaries: string[];
+  /** The values of the margins. Must contains one more element than the boundaries
+    Can be a percentage `X%` or a time in minutes per 100 kilometer `Xmin/100km` */
+  values: string[];
+};
+export type PathItem = PathItemLocation & {
+  /** Metadata given to mark a point as wishing to be deleted by the user.
+    It's useful for soft deleting the point (waiting to fix / remove all references)
+    If true, the train schedule is consider as invalid and must be edited */
+  deleted?: boolean;
+  id: string;
+};
+export type PowerRestrictionItem = {
+  from: string;
+  to: string;
+  value: string;
+};
+export type ScheduleItem = {
+  /** The expected arrival time at the stop.
+    This will be used to compute the final simulation time. */
+  arrival?: string | null;
+  at: string;
+  /** Whether the schedule item is locked (only for display purposes) */
+  locked?: boolean;
+  reception_signal?: ReceptionSignal;
+  /** Duration of the stop.
+    Can be `None` if the train does not stop.
+    If `None`, `reception_signal` must be `Open`.
+    `Some("PT0S")` means the train stops for 0 seconds. */
+  stop_for?: string | null;
+};
+export type PathAndScheduleChangeGroup = {
+  margins: Margins;
+  path: PathItem[];
+  power_restrictions: PowerRestrictionItem[];
+  schedule: ScheduleItem[];
+};
+export type RollingStockChangeGroup = {
+  category?: TrainCategory | null;
+  comfort: Comfort;
```

Comment:
If the occurrence is an exception because of its `category`, we would need the possibility not to put a `comfort` here ?
```suggestion
  comfort?: Comfort;
```

Reply:
Maybe we can do an new `silo|change group` for rolling stock category ?